<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="CISPL-IO-80" failures="1" tests="5" name="Home Page Test" time="28.383" errors="0" timestamp="2024-07-31T19:46:22 IST">
  <testcase classname="TestPackage.HomeTest" name="categorylinkTest" time="0.635"/>
  <testcase classname="TestPackage.HomeTest" name="heading" time="0.457"/>
  <testcase classname="TestPackage.HomeTest" name="homecontent" time="0.543">
    <skipped/>
  </testcase> <!-- homecontent -->
  <testcase classname="TestPackage.HomeTest" name="homecontent" time="0.629">
    <skipped/>
  </testcase> <!-- homecontent -->
  <testcase classname="TestPackage.HomeTest" name="homecontent" time="0.45">
    <failure type="java.lang.AssertionError" message="expected [If you have questions, see the Documentation, or post in the Forums at nopCommerce.com 1] but found [If you have questions, see the Documentation, or post in the Forums at nopCommerce.com]">
      <![CDATA[java.lang.AssertionError: expected [If you have questions, see the Documentation, or post in the Forums at nopCommerce.com 1] but found [If you have questions, see the Documentation, or post in the Forums at nopCommerce.com]
at TestPackage.HomeTest.homecontent(HomeTest.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- homecontent -->
</testsuite> <!-- Home Page Test -->
